src/relay/pass/gradient.cc

```diff
@@ -348,11 +348,16 @@ struct ReverseAD : ExprMutator {
           args.push_back(ll->Push(VisitExpr(arg)));
         }
         std::vector<Expr> orig_args;
-        for (size_t i = 0; i < args.size(); ++i) {
-          orig_args.push_back(GetValue(op->args[i]->checked_type(), args[i], ll));
+        for (size_t i = 0; i < args.size(); i++) {
```

Comment:
this change make sense, but I had changed gradient. Can you read the change and apply it there? It isnt too hard to understand.